iothub-digitaltwins-timeseriesinsights/create-solution.sh

```diff
@@ -0,0 +1,174 @@
+    export EVENTS_PER_SECOND=100
+    export SIMULATOR_INSTANCES=1
+fi
+
+# last checks and variables setup
+if [ -z ${SIMULATOR_INSTANCES+x} ]; then
+    usage
+fi
+
+export RESOURCE_GROUP=$PREFIX
+
+# remove log.txt if exists
+rm -f log.txt
+
+echo "Checking pre-requisites..."
+
+source ../assert/has-local-az.sh
+source ../assert/has-local-jq.sh
+source ../assert/has-local-dotnet.sh
+source ../assert/has-local-terraform.sh
+
+echo
+echo "Streaming at Scale with Azure Digital Twins and Azure Time Series Insights"
+echo "=========================================================================="
+echo
+
+echo "Steps to be executed: $STEPS"
+echo
+
+echo "Configuration: "
+echo ". Resource Group  => $RESOURCE_GROUP"
+echo ". Region          => $LOCATION"
+echo ". IoT Hub         => SKU: $TF_VAR_iothub_sku, Capacity: $TF_VAR_iothub_capacity"
+echo ". Function        => SKU: $TF_VAR_function_sku, Workers: $TF_VAR_function_workers"
+echo ". TS Insights     => PAYG"
+echo ". Simulators      => $SIMULATOR_INSTANCES"
+echo
+
+echo "Deployment started..."
+echo
+
+echo "***** [I] Deploying INFRASTRUCTURE"
+
+    RUN=`echo $STEPS | grep I -o || true`
+    if [ ! -z "$RUN" ]; then
+        terraform init
+        terraform plan -var appname=$RESOURCE_GROUP -var resource_group=$RESOURCE_GROUP -var location=$LOCATION -out tfplan
+        terraform apply tfplan
+    fi
+echo
+
+echo "***** [D] Setting up DATA"
+
+    RUN=`echo $STEPS | grep D -o || true`
+    if [ ! -z "$RUN" ]; then
+        iothub_registry_write_primary_connection_string=$(terraform output -raw iothub_registry_write_primary_connection_string)
+        digital_twins_service_url=$(terraform output -raw digital_twins_service_url)
+        time_series_insights_data_access_fqdn=$(terraform output -raw time_series_insights_data_access_fqdn)
+
+        CONNECTION_STRING="$iothub_registry_write_primary_connection_string" dotnet run -p src/PopulateIoTHub
+        dotnet run -p src/PopulateDigitalTwinsModel "$digital_twins_service_url" "models/digital_twin_types.json"
+        dotnet run -p src/PopulateTimeSeriesInsightsModel "$time_series_insights_data_access_fqdn" "models/time_series_insights_types.json" "models/time_series_insights_hierarchies.json"
+    fi
+echo
+
+echo "***** [T] Starting up TEST clients"
+
+    RUN=`echo $STEPS | grep T -o || true`
+    if [ ! -z "$RUN" ]; then
+        export SIMULATOR_CONNECTION_SETTING=IotHubConnectionString="$(terraform output -raw iothub_telemetry_send_primary_connection_string)"
+        source ../simulator/run-simulator.sh
+    fi
+echo
+
+echo "***** [M] Starting METRICS reporting"
+
+    RUN=`echo $STEPS | grep M -o || true`
+    if [ ! -z "$RUN" ]; then
+        export IOTHUB_RESOURCES=$(terraform output -raw iothub_resource_id)
+        export EVENTHUB_NAMESPACES=$(terraform output -raw eventhub_namespace_names)
+        source ../components/azure-event-hubs/report-throughput.sh
+    fi
+echo
+
+echo "***** [V] Starting deployment VERIFICATION"
```

Comment:
What is this step doing exactly? I could not find any information on this step in the README.

Reply:
Added comment in README